terraform/modules/splunk-server/resources.tf

```diff
@@ -48,7 +48,7 @@ resource "aws_instance" "splunk-server" {
+    command = "ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -u ubuntu --private-key ${var.config.private_key_path} -i '${aws_instance.splunk-server.public_ip},' playbooks/splunk_server.yml -e 'ansible_python_interpreter=/usr/bin/python3 splunk_admin_password=${var.config.attack_range_password} splunk_url=${var.config.splunk_url} splunk_binary=${var.config.splunk_binary} s3_bucket_url=${var.config.s3_bucket_url} splunk_escu_app=${var.config.splunk_escu_app} splunk_asx_app=${var.config.splunk_asx_app} splunk_windows_ta=${var.config.splunk_windows_ta} splunk_cim_app=${var.config.splunk_cim_app} splunk_sysmon_ta=${var.config.splunk_sysmon_ta} splunk_python_app=${var.config.splunk_python_app} splunk_mltk_app=${var.config.splunk_mltk_app} caldera_password=${var.config.attack_range_password} install_es=${var.config.install_es} splunk_es_app=${var.config.splunk_es_app} phantom_app=${var.config.phantom_app} phantom_server=${var.config.phantom_server} phantom_server_private_ip=${var.config.phantom_server_private_ip} phantom_admin_password=${var.config.attack_range_password} splunk_security_essentials_app=${var.config.splunk_security_essentials_app} splunk_bots_dataset=${var.config.splunk_bots_dataset} punchard_custom_visualization=${var.config.punchard_custom_visualization} status_indicator_custom_visualization=${var.config.status_indicator_custom_visualization} splunk_attack_range_dashboard=${var.config.splunk_attack_range_dashboard} timeline_custom_visualization=${var.config.timeline_custom_visualization} splunk_stream_app=${var.config.splunk_stream_app} splunk_zeek_ta=${var.config.splunk_zeek_ta} splunk_server_private_ip=${var.config.splunk_server_private_ip} install_mltk=${var.config.install_mltk} range_name=${var.config.range_name}'"
```

Comment:
what was changed here? 
@P4T12ICK we need to break this into multiple lines, otherwise is hard to track changes.

Reply:
The variable range_name at the end. If terraform supports it, we can do it.

Reply:
Yes, the range_name variable is added to the attack_range.conf file, but I wanted to pass it to Ansible as a variable in order to use it in configurations.  For instance, in a Phantom Ansible Playbook I would use it to set the instance name instead of a random name like squash-triple-red.

Reply:
Sorry to come this late to the party. I was checking in this PR, and the variable that you are passing, seems that is not being currently used. Perhaps it's not worth passing it if it's not used, but if you need it add it in a future pr.